cookies/abtest_test.go

```diff
@@ -0,0 +1,76 @@
+package cookies
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUnitABTest(t *testing.T) {
+
+	var testDomain = "www.ons.gov.uk"
+	var testTime = time.Date(2009, 11, 17, 20, 34, 58, 651387237, time.UTC)
+	var testServices = ABServices{NewSearch: &testTime}
+	var expectedEscapedCookieValue = "%7B%22new_search%22%3A%222009-11-17T20%3A34%3A58.651387237Z%22%7D"
+	var testMarshalledValue, _ = json.Marshal(testServices)
```

Comment:
Even though it is test it is good to handle err and bomb out of test with failure.

```suggestion
	testMarshalledValue, err := json.Marshal(testServices)
	if err != nil {
	    t.Fatalf("failed to marshal test data", err)
	}
```

See: https://pkg.go.dev/testing#B.Fatalf

Reply:
👍